app/api/dao/user.py

```diff
@@ -54,17 +97,19 @@ def create_user(data):
 
         return messages.USER_WAS_CREATED_SUCCESSFULLY, 200
 
+    # checks if the password is secure (Password security validation function)
+
     @staticmethod
     @email_verification_required
     def delete_user(user_id):
         """ Deletes a user.
-        
```

Comment:
can you change your on save lint so it will not doing unnecesary changes?

Reply:
@annabauza 
Okay I will change that.